artifacts/Products/ShowProducts.recipes

```diff
@@ -13,17 +13,25 @@ particle ShowProduct in 'source/ShowProduct.js'
   in Product product
   consume item
 
-recipe MuxedProductItem &muxProduct
+recipe MuxedItem &muxItem
   ItemMultiplexer
 
+// TODO(mmandlis): List + &muxItem should be a generic recipe used by both -
+// show products and show wishlist recipes. 
 recipe ProductList &showShopList
-  copy #shoplist as newlist
+  copy #shoplist as shoplist
   List
-    items = newlist
+    items <- shoplist
+  &muxItem
+    * <- shoplist
   description `show ${List.items}`
 
 // Show [person]'s wishlist
 recipe
-  use #claire as person
+  // use #claire as person
```

Comment:
This was setup to ensure (1) only Claire can own the wishlist (2) suggestion to recommend from the wishlist only appears after copying the `Claire` person via another suggestion (to establish relevance).

Are those qualities preserved here, or discarded for other virtues?

Reply:
FWIW right now the "Show wishlist" recipe doesn't resolve - there are no particles in that recipe that bind to the `use #claire as person` handle, so the type of the handle remains unknown.
In order to make is work, we would need to have a particle that fetches person's wishlist, with `in Person` and `out [Product] wishlist`.

Reply:
So, I was reacting to the wrong thing, sorry about that. I mixed up 'ShowProducts.recipe' and 'Recommend.recipes' in my rush to provide feedback.

Having said that, it seems to me that a wishlist needs to be attached to a person. As you say, we don't really have that infrastructure. IMO, this is a signal to either fix it properly, or get rid of the ancillary dependencies (aka 'Show wishlist').

Probably this is a larger discussion that shouldn't be blocking your PR however.

Reply:
Fwiw, if I add `#items` to `ClairesWishlist` store as defined in `Demo/ClairesWishlist.manifest`, I get _Show wishlist_ suggestion for free, without any exports from `ShowProducts.recipes` other than `ShowProduct`